package.json

```diff
@@ -16,6 +16,7 @@
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.6",
     "axios": "^1.1.3",
+    "date-fns": "^2.29.3",
```

Comment:
Pls add date-fns library to the list of software. It's a requirement from the client.
https://mindlogger.atlassian.net/wiki/spaces/MINDLOGGER1/pages/20086829/SOUP+Software+of+Unknown+Provenance